package.json

```diff
@@ -7,6 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "watch": "nodemon index.js",
     "seed": "node seeds/seed.js",
+    "seed_heroku": "dotenv -e .env.jaws -- node seeds/seed.js",
```

Comment:
What goes into .env.jaws?

Reply:
our db variables. you don't need it for the app to work. i need them to restart the db when needed